src/index.js

```diff
@@ -1 +1,50 @@
 import './styles.css';
+import {
```

Comment:
- **[OPTIONAL]** This is not part of the requirements but it's highly recommended for UX  enhancement, it will be greater if the app showed the score list when the page is visited or refreshed, the function for that is already implemented, and well done for that 👏🏼 , so let's say when the event `DOMContentLoaded` is triggered the `fetchScores` method is invoked 👍🏼